.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,7 @@ Note that by _not_ including a description, you are asking reviewers to do extra
 ## Checklist
```

Comment:
Given the conversations we're having around style checks, should we make it something like `Code style checks for new code contributions pass` for now?

Reply:
Will there be code in the initial commit that doesn't adhere to the specified standards and is too much of a lift to fix? Even if that's true, the linting would run against code being committed. If you're making changes to a class, and other code inside that class doesn't meet code style requirements, fixing just that class should be a light effort and, such changes would cumulatively help clean up the code base. Thoughts?

Reply:
This also feels like something we'd want to look into more closely when we're implementing the linting enforcement.

Reply:
Yes, my understanding is that the initial code commit won't adhere to all the standards we want in place. I do think that fixing anything the linter picks up from the changed code in a subsequent PR is reasonable to expect someone to fix, and that's what I was trying to suggest with this language change. Maybe "new code contributions" is not the clearest way to communicate this, so I'd say go with what makes it most clear from your POV.
Agreed that we can double back to this when we work on linting enforcement to ensure it's accurate.

Reply:
Assuming we configure linting checks correctly, then, we should be able to leave the text as is. If lints are only run against newly contributed code, or against classes that have changed, then "all lint checks pass" still works IMO.